projects/kit/components/accordion/accordion-item/accordion-item-eager-content.directive.ts

```diff
@@ -0,0 +1,16 @@
+import {ChangeDetectorRef, Directive, Inject, Self, TemplateRef} from '@angular/core';
+import {PolymorpheusTemplate} from '@tinkoff/ng-polymorpheus';
+
+@Directive({
+    selector: '[tuiAccordionItemEagerContent]',
+})
+export class TuiAccordionItemEagerContentDirective extends PolymorpheusTemplate<{}> {
```

Comment:
If it's eager content, there's no need to extend template, it will not be a template, you would put it on a plain DOM element.

Reply:
Yeah, this make sense, thank you